### ISSUE TITLE
ArgumentError in tutorial on Hidden Markov Models

### ISSUE THREAD

**Issue by smonsays** [TuringLang/Turing.jl#1411]
I tried running the [tutorial on Hidden Markov Models]( https://github.com/TuringLang/Turing.jl/edit/master/docs/_tutorials/4_BayesHmm.md) but encountered `ERROR: ArgumentError: index m not found` when running the following lines:
```julia
# Extract our m and s parameters from the chain.
m_set = c[:m].value.data
s_set = c[:s].value.data
```
I am not sure how this can be fixed, i.e. how does one access a set of parameters that belong to the same array in a chain object?

**Comment by devmotion** [comment]
You can use `namesingroup` or `group`, see https://github.com/TuringLang/MCMCChains.jl#groups-of-parameters. I guess it is also more natural to use `Array` to obtain an array ouput, `.value` is an implementation detail (see https://github.com/TuringLang/MCMCChains.jl#exporting-chains).

**Comment by smonsays** [comment]
Thanks, the following works:
```julia
m_set = Array(group(c, :m))
s_set = Array(group(c, :s))
```
There is still an issue with the plotting, but I think this will now generally be addressed in [TuringLang/TuringTutorials#86.](https://github.com/TuringLang/TuringTutorials/issues/86)
